xlsxwriter/worksheet.py

```diff
@@ -804,10 +804,6 @@ def write_url(self, row, col, url, cell_format=None,
         # Strip the mailto header.
         string = string.replace('mailto:', '')
 
-        # Check that row and col are valid and store max and min values
```

Comment:
`write_url()` can be called by the user so this check is necessary.

Reply:
roger, will fix.